src/ad/azext_ad/generated/_help.py

```diff
@@ -0,0 +1,148 @@
+# --------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+#
+# Code generated by Microsoft (R) AutoRest Code Generator.
+# Changes may cause incorrect behavior and will be lost if the code is
+# regenerated.
+# --------------------------------------------------------------------------
+# pylint: disable=too-many-lines
+
+from knack.help_files import helps
+
+
+helps['ad ds'] = """
+    type: group
+    short-summary: Manage domain service with ad
```

Comment:
The `short-summary` should usually ends with a period (`.`), like

https://github.com/Azure/azure-cli/blob/84e0e45de5f99936ca5b933493fd6234dbf648ae/src/azure-cli/azure/cli/command_modules/vm/_help.py#L67

Reply:
Auto-generated by CodeGen

Reply:
the description is not easy to understand with `ad`.

Reply:
Refined in `manual/_help.py`